monitoring/quickstart.rb

```diff
@@ -0,0 +1,40 @@
+require "google/cloud/monitoring"
+
+def quickstart
+  # [START monitoring_quickstart]
+
+  # TODO: Replace Google Cloud Platform project ID
```

Comment:
What is the plan for completing this TODO? Any way to do it in this PR?

Reply:
It was just for information. Removed now.